src/components/flowchart/draw.js

```diff
@@ -134,15 +145,14 @@ export const drawNodes = function (changed) {
     .merge(exitNodes)
     .filter((node) => typeof node !== 'undefined');
 
-  if (changed('nodes')) {
+  if (changed('nodes', 'newParamsFlag')) {
     enterNodes
       .attr('tabindex', '0')
       .attr('class', 'pipeline-node')
       .attr('transform', (node) => `translate(${node.x}, ${node.y})`)
       .attr('data-id', (node) => node.id)
-      .classed(
-        'pipeline-node--parameters',
-        (node) => node.type === 'parameters'
+      .classed('pipeline-node--parameters', (node) =>
+        newParamsFlag ? node.type === 'parameters' : null
```

Comment:
This class looks like it already existed, so might be able to revert the flag check unless its usage changed?

Reply:
Hi, yes so this class did exist before but it wasn't really used for anything. Now this class adds a stroke of the accent colour to highlights parameters when they are enabled. 
Of course, this feature is not available if newParamsFlag = false :D